modules/gdscript/gdscript_warning.h

```diff
@@ -78,6 +78,7 @@ class GDScriptWarning {
 		EMPTY_FILE, // A script file is empty.
 		SHADOWED_GLOBAL_IDENTIFIER, // A global class or function has the same name as variable.
 		INT_ASSIGNED_TO_ENUM, // An integer value was assigned to an enum-typed variable without casting.
+		STATIC_CALLED_ON_INSTANCE,
```

Comment:
Should have a comment like the others I suppose.

Reply:
Done